source/includes/_tipser-widget.md

```diff
@@ -145,6 +145,58 @@ A complete index of all the supported configuration options can be found in [All
 It is possible to use Tipser Elements in test (staging) environment to be able to do test checkouts. The environment can be activated using the `env` configuration option, as in the snippet, 
 where supported environments are `stage` and `prod`. This configuration option is optional and in case it is missing, `prod` environment will be used. 
 
+## Adding event handlers ##
+Event handlers may be passed as a part of configuration. There is a number of events exposed for developer. Each of the handler may have a different params as arguments of handler. 
```

Comment:
`Each of the handler may have a different params as arguments of handler.` - not sure if this sentence is needed. It should be evident from the docs of specific events.